src/ncp/spinel.h

```diff
@@ -2695,6 +2696,58 @@ typedef enum
 
     SPINEL_PROP_MESHCOP__END = 0x90,
 
+    SPINEL_PROP_SERVER__BEGIN = 0x90,
+
+    /// Server Allow Local Network Data Change
+    /** Format `b` - Read-write
+     *
+     * Required capability: SPINEL_CAP_THREAD_SERVICE
+     *
+     * Set to true before changing local server net data. Set to false when finished.
+     * This allows changes to be aggregated into a single event.
+     *
+     */
+    SPINEL_PROP_SERVER_ALLOW_LOCAL_DATA_CHANGE = SPINEL_PROP_SERVER__BEGIN + 0,
```

Comment:
Can you please add the new properties to `spinel_prop_key_to_cstr()` in `spinel.c`? Thanks.